### DISCUSSION TITLE
New STT Plugin - Whisper

### DISCUSSION THREAD

**Discussion by JarbasAl** [OpenVoiceOS/ovos-buildroot#155] [Show and tell]
https://github.com/OpenVoiceOS/ovos-stt-plugin-whisper

**Comment by JarbasAl** [comment]
@j1nx

**Comment by j1nx** [comment]
Will have a look. Which whisper inference method / library does this use?

**Reply by JarbasAl** [reply]
openai-whisper

**Reply by j1nx** [reply]
Nice, will check it out.

could you also create a lite version based on this code?
https://github.com/fquirin/speech-recognition-experiments/blob/main/whisper-tflite/test.py

At the moment only this model is supported;
https://github.com/usefulsensors/openai-whisper/blob/main/models/whisper-tiny.en.tflite

But it is fast😎

**Reply by JarbasAl** [reply]
https://github.com/OpenVoiceOS/ovos-stt-plugin-whisper-tflite

**Comment by mikejgray** [comment]
This is done, right?

**Reply by JarbasAl** [reply]
yeah this is an announcement in show and tell :P
